examples/phys/FlatteLineShape.inl

```diff
@@ -350,7 +350,7 @@ int main(int argv, char** argc)
 
     double f0_MASS        = 0.965;
 	double f0_MAG		  = 12.341;
-	double f0_Phase		  = -62.852*(M_PI / 180) ;
+	double f0_Phase		  = -62.852*(M_PI / 180) +M_PI;
```

Comment:
The primary documentation of code, is the code itself. So, please enforce double precision syntax where you are using it. For example '''180''' -> '''180.0'''